web/src/SourcegraphWebApp.tsx

```diff
@@ -242,6 +246,18 @@ class ColdSourcegraphWebApp extends React.Component<SourcegraphWebAppProps, Sour
                 }
             })
         )
+
+        this.subscriptions.add(
+            from(this.platformContext.settings).subscribe(settingsCascade => {
+                const splitSearchModes: boolean =
```

Comment:
Nit: `boolean` should just be inferred?

```suggestion
                const splitSearchModes =
```